installer/client/src/pages/ConfirmationScreen.vue

```diff
@@ -51,6 +51,25 @@
       OrangeHRM Admin Username
     </oxd-text>
     <oxd-text class="confirmation-text-info">{{ adminUserName }}</oxd-text>
+    <br v-if="database.enableDataEncryption" />
+    <oxd-text
+      v-if="database.enableDataEncryption"
+      class="confirmation-text-section"
+      >Data Encryption</oxd-text
+    >
+    <oxd-text
+      v-if="database.enableDataEncryption"
+      class="confirmation-text-info"
+    >
+      Data Encryption is on. Employee Social Security Number and Employee Basic
+      Salary would be encrypted.
```

Comment:
```suggestion
      Data Encryption Is On. Employee Social Security Number and Employee Basic
      Salary Would Be Encrypted.
```
should be in either sentence case or title case :thinking:

Reply:
changed to `Data Encryption Is On. Employee Social Security Number and Employee Basic
      Salary Will Be Encrypted.`